java/http/java-http-consumer/pom.xml

```diff
@@ -52,8 +52,12 @@
             <groupId>io.opentelemetry</groupId>
             <artifactId>opentelemetry-semconv</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>3.4.1</version>
+        </dependency>
```

Comment:
I see you are adding this Kafka dependency just for getting the constant for the `enable.auto.commit` configuration property. Tbh I would avoid that, just for a constant. Declare your own in the code and remove the dependency I would say.

Reply:
Done